### ISSUE TITLE
ManagementGrain.GetTotalActivationCount() does not return a result with 2.4.0

### ISSUE THREAD

**Issue by SebastianStehle** [dotnet/orleans#5857]
Hi,

I updated my application to 2.4.0 and with a real membership provider like MongoDB I see a few behaviors:

1. The dashboard does not show any results.

2. GetTotalActivationCount() returns 0

3. GetTotalActivationCount() something throws an exception:

```
message:  "Object reference not set to an instance of an object."    
   stackTrace:  "   at Orleans.Runtime.MembershipService.SystemTargetBasedMembershipTable.ReadAll()
   at Orleans.Runtime.MembershipService.MembershipTableManager.RefreshInternal(Boolean requireCleanup)
   at Orleans.Runtime.MembershipService.MembershipTableManager.Refresh()
   at Orleans.Runtime.Management.ManagementGrain.GetHosts(Boolean onlyActive)
   at Orleans.Runtime.Management.ManagementGrain.GetTotalActivationCount()
   at Orleans.Runtime.OrleansCodeGenManagementGrainMethodInvoker.Invoke(IAddressable grain, InvokeMethodRequest request)
   at Orleans.Runtime.GrainMethodInvoker.Invoke()
   at OrleansDashboard.Metrics.GrainProfilerFilter.Invoke(IIncomingGrainCallContext context)
   at Orleans.Runtime.GrainMethodInvoker.Invoke()
   at Squidex.Infrastructure.Orleans.LoggingFilter.Invoke(IIncomingGrainCallContext context) in /src/src/Squidex.Infrastructure/Orleans/LoggingFilter.cs:line 30"    
   type:  "System.NullReferenceException"    
```

I have pushed a small example to: https://github.com/OrleansContrib/OrleansDashboard/tree/mongo-example/Tests/TestHosts/TestHostWithMongoClustering

**Comment by benjaminpetit** [comment]
I do not know the consul provider very well, but from ``MongoMembershipCollection``

``` csharp
[…]
// MongoDB does not support the extended Membership Protocol and will always return the same table version information
private static readonly TableVersion tableVersion = new TableVersion(0, "0");
[…]
```

Will definitely cause issues in 2.4.0. I am not sure this is what you are experiencing though. Still looking into it.

**Comment by benjaminpetit** [comment]
I do not repro the `NullReferenceException`, but I can say that all provider that do not have an atomic increasing TableVersion, will not work with 2.4.0

**Comment by SebastianStehle** [comment]
I have seen the exception only once, 

I thought that it is a minor release without breaking changes

**Comment by SebastianStehle** [comment]
DynamoDB also has a static table version.

**Comment by SebastianStehle** [comment]
Fast response... awesome :)

**Comment by sergeybykov** [comment]
We just published v2.4.1 that includes #5863 to to nuget.org.

**Comment by sergeybykov** [comment]
Resolved via #5863.
